cdc/owner.go

```diff
@@ -299,10 +302,23 @@ func (o *Owner) balanceTables(ctx context.Context) error {
 
 func (o *Owner) flushChangeFeedInfos(ctx context.Context) error {
 	snapshot := make(map[model.ChangeFeedID]*model.ChangeFeedStatus, len(o.changeFeeds))
+	minCheckpointTs := uint64(math.MaxUint64)
 	for id, changefeed := range o.changeFeeds {
```

Comment:
what happened if it doesn't exist changefeed

Reply:
Great catch.